Description: Initialize variable 'route_py_dict'
 gcc (v4.4.5 20100816) emits warning about it when CFLAGS+="-O2".
 .
 For assurance, variable 'route_py_callbase' is also initialized.
Author: Leonid Borisenko <leo.borisenko@gmail.com>

Index: uwsgi-0.9.6/routing.c
===================================================================
--- uwsgi-0.9.6.orig/routing.c	2010-08-30 02:43:48.000000000 +0300
+++ uwsgi-0.9.6/routing.c	2010-08-30 02:44:19.000000000 +0300
@@ -63,7 +63,7 @@
 void uwsgi_route_action_wsgi(struct uwsgi_server *uwsgi, struct wsgi_request *wsgi_req, struct uwsgi_route *ur) {
 
 	int i;
-	PyObject *route_py_callbase, *route_py_dict;
+	PyObject *route_py_callbase = NULL, *route_py_dict = NULL;
 
 	uwsgi_log("managing WSGI route...\n");
         if (ur->callable == NULL) {
@@ -106,7 +106,7 @@
 
 void uwsgi_route_action_uwsgi(struct uwsgi_server *uwsgi, struct wsgi_request *wsgi_req, struct uwsgi_route *ur) {
 
-	PyObject *route_py_callbase, *route_py_dict;
+	PyObject *route_py_callbase = NULL, *route_py_dict = NULL;
 	int i;
 
 	uwsgi_log("managing uwsgi route...\n");
